### PR TITLE
Feat: recent txs, overview for sequencer tier

### PR DESCRIPTION
**Pull request by evilpeach**
Fixes CFE-532, CFE-536

What I've done
- Recent Txs for Sequencer tier
- Overview for Sequencer tier (avg_blocktime, total_txs)
- Support IBC, Opinit badges
- Introduce `handleQueryByTier` and apply to `useOverviewsStats` as example.

### CODE REVIEW

File: src/lib/layout/InformationFooter.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import type { IconKeys } from "lib/components/icon";
 import { CustomIcon } from "lib/components/icon";
 import { USER_GUIDE_DOCS_LINK } from "lib/data";
 import { useLatestBlockLcd } from "lib/services/block";
-import { useOverviewsStats } from "lib/services/overviewService";
+import { useOverviewsStats } from "lib/services/stat";
```
Comment (songwongtp) [inline src/lib/layout/InformationFooter.tsx:11]: `lib/services/stats`

File: src/lib/pages/home/sequencer.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+          />
+          <CardInfo
+            title={blockTimeInfo.title}
+            tooltip={blockTimeInfo.tooltip}
+            value={overviewsStats?.blockTime.toFixed(3).concat("s")}
+            isLoading={isLoading}
+            navigate={toTxs}
+          />
+        </Flex>
+      </Flex>
```
Comment (songwongtp) [inline src/lib/pages/home/sequencer.tsx:108]: should we move to like `components/HomeTop.tsx`?

File: src/lib/services/stat/sequencer.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import type { OverviewsStats } from "../types";
+import { zOverviewsStatsResponseSequencer } from "../types";
+import { parseWithError } from "lib/utils";
+
+export const getOverviewStatsSequencer = async (
+  endpoint: string,
+  chainId: string
+): Promise<OverviewsStats> => {
+  return Promise.all([
```
Comment (songwongtp) [inline src/lib/services/stat/sequencer.ts:11]: implicit return

File: src/lib/services/types/stat.ts
Comment (songwongtp) [inline src/lib/services/types/stat.ts:1]: stats.ts

**Review by songwongtp — APPROVED** [review]

**Review by Poafs1 — APPROVED** [review]

File: src/lib/pages/home/sequencer.tsx (hunk 2 above)
Comment (evilpeach) [reply src/lib/pages/home/sequencer.tsx:108]: Let's do in the next ticket.

**Pull request merged**